.git-blame-ignore-revs

```diff
@@ -0,0 +1,5 @@
+# .git-blame-ignore-revs
+# Exclude formatting commits made by black
+a9facdf30ca467b6e16d13ce88d8889bba7f2e1c
+ea81ba259216b8be7fb86ef5d3e9c151a94edf87
+81fe44f0dd9785b1665bbbea047aed591912534b
```

Comment:
```suggestion
a9facdf30ca467b6e16d13ce88d8889bba7f2e1c   # more black formatting Aug 17, 2021
ea81ba259216b8be7fb86ef5d3e9c151a94edf87   # formatting Aug 16, 2021 
81fe44f0dd9785b1665bbbea047aed591912534b  # apply black Aug 11, 2021
```

Suggesting that we add a timestamp to aid in quickly assessing how much blame ignore coverage we have